requirements.txt

```diff
@@ -13,3 +14,4 @@ pytest-cov
 Sphinx
 sphinx-issues
 twine
+zopfli
```

Comment:
how does this relate to the 3.x update? are those dependencies really necessary? zopfli, for example, is not packaged in debian and will slow down inclusion a little...

Reply:
Not related, I just use this file to track optional dependencies. zopfli/brotli were added recently for the compress_assets plugin (#300), but it can also work with gzip.